src/cljs/athens/electron.cljs

```diff
@@ -323,6 +324,55 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+  (def schemaTest
+  {:schema/version {}
+   :block/uid      {:db/unique :db.unique/identity}
+   :node/title     {:db/unique :db.unique/identity}
+   :attrs/lookup   {:db/cardinality :db.cardinality/many}
+   :block/children {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}
+   :block/refs     {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}}
+  )
+  
+  ;; Create an empty Db for test 
+  (def test-db
+    (-> (d/empty-db schemaTest))
+  )
+  
+  ;; Update index.transit
+  (defn writeDbIndex [file filepath]
+    (let [w  (.. fs (writeFile filepath file (fn [e](dispatch [:boot/desktop]))))])
+  )
+  
+  
+  (defn open-dialog-index
+    "Allow user to open a Backup file."
+    [filepath]
+    (js/alert "Your Data file is corrupted or incorrect, Please select a Backup file")
```

Comment:
please fix capitalization, capital letters belong to things like proper nouns and start of sentence.

"Your data file is corrupted or incorrect, please select a backup file."

Reply:
done